docs/topics/proc_running-the-secured-booster-integration-tests-on-vx.adoc

```diff
@@ -0,0 +1,29 @@
+[#running-the-secured-booster-integration-tests_context]
+= Running the {name-mission-secured} Booster Integration Tests
+
+.Prerequisites
+
+* The `oc` client authenticated.
+// * An empty OpenShift project in which to run the tests.
+
+.Procedure
+// space here to prevent the section title and admonition from sticking together in the rendered version
+{nbsp}
+include::note-integration-tests-app-deletion-warning.adoc[]
+
+By default, the SSO server is deployed (and destroyed) as part of testing. The steps for executing integration tests are as follows:
+
+. In a terminal application, navigate to the directory with your project.
+. Execute the integration tests:
++
+[source,bash,options="nowrap",subs="attributes+"]
+--
+mvn clean verify -Popenshift,openshift-it
+--
+
+You can also skip the SSO server deployment in case you have deployed a SSO server instance beforehand, e.g. by executing `oc create -f service.sso.yaml`.
```

Comment:
This sentence could probably use some rewording. I know that "e.g. by executing `oc create -f service.sso.yaml`" is _meant to_ bind to "you have deployed a SSO server instance beforehand", but I can see how people can misinterpret that and think that it binds to "you can skip the SSO server deployment".

Something like this might be better:

> If you deployed an SSO server beforehand, e.g. by executing `oc create -f service.sso.yaml`, set the system property `skip.sso.init` to `true` when running the test.
> The test will then use the existing SSO server and won't deploy its own.
> mvn clean verify -Popenshift,openshift-it -Dskip.sso.init=true